waku/waku_archive/driver/sqlite_driver/queries.nim

```diff
@@ -61,14 +58,6 @@ proc queryRowPubsubTopicCallback(
 
   return pubsubTopic
 
-proc queryRowDigestCallback(s: ptr sqlite3_stmt, digestCol: cint): seq[byte] =
-  let
-    digestPointer = cast[ptr UncheckedArray[byte]](sqlite3_column_blob(s, digestCol))
-    digestLength = sqlite3_column_bytes(s, digestCol)
-    digest = @(toOpenArray(digestPointer, 0, digestLength - 1))
-
-  return digest
-
```

Comment:
Why we can remove that one?

Reply:
Digest is not a concept we use in v3. It is redundant with message hashes.